### ISSUE TITLE
Storybook components are being intercepted

### ISSUE THREAD

**Issue by michaelryancaputo** [infinitered/reactotron#168]
Looks like I can't access (at least not directly) my components that are built with react-storybook.

Would be great to be able to identify that the user is using storybook, and be able to use Reactotron in that context.

**Comment by skellock** [comment]
Oh?  When you say components, do you mean there's state data in there?

I'd love to help out somehow.  Storybook looks amazing.

**Comment by michaelryancaputo** [comment]
Yeah, i'm using storybook to build out a component that contains quite a bit of stuff (including redux actions) - (the reason i'm doing this is because we're slowly migrating off of angular).

**Comment by skellock** [comment]
I'll take a closer look at storyboards.  i thought they were just a hosting container for views.  This is news to me.  

So does it have its on separate redux store then?

**Comment by amna-geniteam** [comment]
Need urgent help: 
I have made react components using react-storybook. When i try to import some component in another project, it says: 
ERROR in ./~/device-explorer-ui-kit/src/web/components/Graph/Slider/GraphSlider.js
Module parse failed: D:\BRD\node_modules\mozilla-neo\node_modules\eslint-loader\index.js!D:\BRD\node_modules\device-explorer-ui-kit\src\web\compone
nts\Graph\Slider\GraphSlider.js Unexpected token (31:12)
You may need an appropriate loader to handle this file type.

In short i am unable to use my components in my actual project. help will be highly appreciated.

**Comment by amna-geniteam** [comment]
@arunoda if you can please help.

**Comment by arunoda** [comment]
@amna-geniteam I'm pretty confused here.
Usually we use storybook inside a React library or a project.
So, you could publish that library into NPM, Git or anywhere you like and use in your projects.
Storybook is only a dev tool.
